### PR TITLE
[Bounty] Add BCHN lead maintainer report 2020-06-15

### PR DESCRIPTION
**Pull request by ChaituVR**
Part of #66

### CODE REVIEW

File: blog/BCHN-lead-maintainer-report-2020-06-15.md [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+This was expected to some degree: when there is not a burning issue, miners signal less willingly for particular clients that seek to address the issue.
+
+The IFP hot button issue has passed and BCH has been safely upgraded without a chain split. Subsequently, the percentage of miners signaling for us has gone down from an all time high of almost 17% to a steady state comprising our most steadfast supporters.
+
+We are thankful to those miners who have supported us during the run-up to May and helped to keep Bitcoin Cash free of a forced levy.
+
+Our special thanks to those who continue to signal support for our software using `/BCHN/` in their coinbase strings. Thanks [bitcoin.com](https://www.bitcoin.com/) , Hashpipe and P2Pool.
+
+<figure class="text-center">
+    <img src="https://rcimg.net/images/a85cfb07/3bneA5i1D0usLADh1f7AAIro3asfIlIK1AUSOFhU.png" class="rounded img-fluid" alt="" title="">
```
Comment (ftrader) [inline blog/BCHN-lead-maintainer-report-2020-06-15.md:76]: Please do not use readcash for image hosting.

You already import the images into the ` app/static/img/newsroom/2020-06-15/` folder, please make use of them. I will mark other instances of this review in this file by comments `Use locally hosted image`.

File: blog/BCHN-lead-maintainer-report-2020-06-15.md [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+    <img src="/static/img/newsroom/2020-06-15/lR0FGkOAhTyIFTxBaH4JWgtTww5EyLee2YaYD4eU.png" class="rounded img-fluid" alt="" title="">
+</figure>
+
+We need to ramp those up further (to make dents in yellow and red lines and bump up the green line).
+
+The slope of the increasing red line indicates rate at which backports from upstream (in this case, ABC) are heading towards us. They then get sorted into various categories (planned, deferred, rejected) and planned ones get worked on until they are integrated (merged).
+
+Internal project discussions were in May and held and concluded that the backporting effort would, for now, be shared by existing contributors to the project. This may change at a later date, but two BCHN developers have firmly committed time to this activity, while another developer has flexibly contributed to it. This is in accordance with our Flipstarter proposal, where we did foresee that
+
+> we may either share it across developers or contract further development resources with these funds.
```
Comment (ftrader) [inline blog/BCHN-lead-maintainer-report-2020-06-15.md:132]: Another quoted paragraph which is badly formatted in the rendered HTML.

Please take similar action as to you will with the previous such instance.

File: blog/BCHN-lead-maintainer-report-2020-06-15.md [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+## [Performance optimizations to JSON library and RPC interface](#performance-optimizations-to-json-library-and-rpc-interface)
+
+BCHN developers continue to improve the performance of JSON RPC, chiefly through optimization of the Univalue library, but not limited to that.
+
+This work is driven by BCHN developers @BigBlockIfTrue and Calin Culianu.
+
+Significant improvements have been achieved since BCHN project start, and since v0.21.2 release. More of these will be available in our next minor release v0.21.3.
+
+Work is progressing toward catching up with the performance of [rapidjson](https://rapidjson.org/), the leading JSON library in the field. Currently UniValue's JSON is slower by a factor of 2. This isn't actually that much difference anymore and we hope to release more benchmark results in the near future to showcase relevant BCHN improvements.
+
```
Comment (ftrader) [inline blog/BCHN-lead-maintainer-report-2020-06-15.md:219]: There are two paragraphs from the original which are completely missing here!

**Review by ftrader — CHANGES_REQUESTED** [review]
Good work so far, I have left a few review comments to be addressed please.

File: blog/BCHN-lead-maintainer-report-2020-06-15.md (hunk 3 above)
Comment (ChaituVR) [reply blog/BCHN-lead-maintainer-report-2020-06-15.md:219]: Thanks @ftrader for finding this... i completely missed those, added them back

**Review by ftrader — APPROVED** [review]

**Comment by ftrader** [comment]
Approved, bounty awarded to @ChaituVR .

**Pull request merged**